vips/vip-187/vip-187-testnet.ts

```diff
@@ -128,7 +143,7 @@ export const vip187Testnet = () => {
         signature: "giveCallPermission(address,string,address)",
         params: [XVSBridgeAdmin_Proxy, "transferBridgeOwnership(address)", NORMAL_TIMELOCK],
       },
-      // FASTTRACK TIMELOCK PERMISSIONS
+      //  FASTTRACK TIMELOCK PERMISSIONS
       {
```

Comment:
I think these functions should be executable only by the Normal timelock, so I wouldn't grant Fast-track and Critical timelocks to execute them:
* setPrecrime(address)
* setTrustedRemoteAddress(uint16,bytes)
* transferBridgeOwnership(address)